src/main/java/mate/jdbc/service/impl/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.ManufactureService;
+
+@Service
+public class ManufacturerServiceImpl implements ManufactureService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return manufacturerDao.get(id).orElseGet(Manufacturer::new);
```

Comment:
```suggestion
        return manufacturerDao.get(id);
```
i don't think you should return empty Manufacturer